environment.yml

```diff
@@ -36,7 +36,6 @@ dependencies:
   - wheel
   - xz
   - zlib
-  - pip:
-    - eventio~=0.20.1
+  - eventio~=0.20.1
```

Comment:
You probably want to go to 0.21 since we implemented new containers needed in current prod4 simulations

Reply:
Ok, I'll update that.  We will eventually want to be able to read prod4 (or later prod5) even with old versions of ctapipe, so it would be nice to be able to set this to something that is future proof.  What pattern would you suggest to support any future version of eventio up to an API change?  Would just `>=0.20.1` work? or do you expect major changes at a certain version (so would also need `<=0.25` or something?) I would like to avoid the problems we had with ctapipe-0.6.2 where it was tied too tightly to eventio-0.11 and we couldn't read new data with it.

Reply:
You might consider releasing the next version not as 0.21, but 1.0.0, so then you can use the first digit for API changes, and second for new features like new data block types, and third for bug fixes

Reply:
Ok,  just could release the 1.0

Reply:
I released v1.0: https://github.com/cta-observatory/pyeventio/releases